lib/marked-renderer.js

```diff
@@ -1,7 +1,17 @@
 const { marked } = require('marked')
 
-/** A custom renderer that avoids wrapping `<img>` tags in `<p>` tags */
+/**
+ * Custom markdown renderer that avoids wrapping `<img>` image
+ * tags (or code with syntax highlighting) in `<p>` paragraphs
+ */
 class DesignSystemRenderer extends marked.Renderer {
+  code (code, language, isEscaped) {
+    return !language
+      ? super.html(code)
```

Comment:
Inserting a new line in Markdown HTML (where the HTML isn't being rendered by Nunjucks) will begin a new "language-less" code block. When this happens we can call `super.html()` and continue output